notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1345_TEA_Vibration_audio_freq_analysis.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from scipy import fftpack
</pre><p>fftpack is a legacy package and is not maintained anymore. I suggest to use the fft library from scipy or from numpy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated !